client/src/components/Footer/index.js

```diff
@@ -36,23 +37,19 @@ function Footer() {
             <p>
               Donations to freeCodeCamp go toward our education initiatives, and
               help pay for servers, services, and staff. You can&nbsp;
-              <Link
-                className='inline'
-                external={true}
-                to='https://freecodecamp.org/donate'
-              >
+              <Link className='inline' to='/donate'>
                 make a tax-deductible donation here
               </Link>
               .
             </p>
           </div>
           <div className='footer-col-1'>
             <div className='col-header'>Our Nonprofit</div>
-            <Link to='https://www.freecodecamp.org/news/about-freecodecamp/'>
+            <Link external={true} to='/news/about-freecodecamp/'>
```

Comment:
This link needs to be updated.

Reply:
@raisedadead, thanks for updating that.
It has been resolved now.